igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/client/model/v3/V3Log.java

```diff
@@ -62,7 +62,12 @@ public boolean isReady() {
     }
     int numberOfParts = logParts.size() - 1;
     V3LogPart lastLogPart = logParts.get(numberOfParts);
-    return numberOfParts == lastLogPart.number && lastLogPart.isFinal();
+    String logContent = getContent();
```

Comment:
This seems like a new change...not reverting part of https://github.com/spinnaker/igor/pull/449 ...  Is this at least sort of, a separate improvement?

Reply:
Correct, this is an improvement. Travis will always end a build (failed or not) like this.